### ISSUE TITLE
[Bug]: [Nightly] Streaming node is not stable and always exits abnormally

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#37573]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: ff00a12
- Deployment mode(standalone or cluster): cluster
- MQ type(rocksmq, pulsar or kafka):    Streaming node
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Streaming node is not stable and always exits abnormally.
link: https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI(new)/detail/master/178/pipeline/151
finish at  7%
link: https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI(new)/detail/master/177/pipeline/140
finish at 8%

grafana link: https://grafana-4am.zilliz.cc/d/uLf5cJ3Ga/milvus2-0?orgId=1&from=1731263607402&to=1731269778831&var-datasource=P1809F7CD0C75ACF3&var-namespace=milvus-ci&var-instance=mdss-master-178-py-n&var-collection=All&var-app_name=milvus

### Expected Behavior

run all the cases

**Comment by chyezh** [comment]
should be fixed by PR: #37564

**Comment by NicoYuan1986** [comment]
fixed. 81141bd
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI(new)/detail/master/179/pipeline/114
